src/fcBindingModel.jl

```diff
@@ -1,5 +1,5 @@
 
-function polyfc_ActV(L0, KxStar, f, Rtot::Array, IgGC::Array, Kav::AbstractMatrix, ActI::Vector)
+function polyfc_ActV(L0, KxStar, f, Rtot::Array, IgGC::Array, Kav::AbstractMatrix, ActI::Vector; Mix = true)
```

Comment:
What does Mix do here? It might be helpful to add documentation of this function, so that others know how to use it.

https://docs.julialang.org/en/v1/manual/documentation/index.html